packages/scroller/test/scroller.test.js

```diff
@@ -139,4 +139,44 @@ describe('vaadin-scroller', () => {
       expect(scroller.getAttribute('overflow')).to.equal('top');
     });
   });
+
+  describe('scroll to', () => {
+    describe('vertical', () => {
+      beforeEach(() => {
+        scroller.scrollDirection = 'vertical';
+        scroller.style.fontSize = '15px';
+        scroller.style.maxHeight = '50px';
+        scroller.style.maxWidth = '50px';
+
+        const div = document.createElement('div');
+        div.textContent = 'Long text that does not fit';
+        scroller.appendChild(div);
+      });
+
+      it('should scroll to the bottom', () => {
+        scroller.scrollToEnd();
+        const scrollTopMax = scroller.scrollHeight - scroller.clientHeight;
+        expect(scroller.scrollTop).to.equal(scrollTopMax);
+      });
+      it('should scroll to the top', () => {
+        scroller.scrollToStart();
+        expect(scroller.scrollTop).to.equal(0);
+      });
+    });
+    describe('horizontal', () => {
```

Comment:
Please add tests for `rtl` direction e.g. see the example here:

https://github.com/vaadin/web-components/blob/fab94595fd5e8c11c0709413251be92994a1daa9/packages/tabs/test/tabs.common.js#L97-L106